### ISSUE TITLE
Add new fonts : Castoro

### ISSUE THREAD

**Issue by tiroj** [google/fonts#2493]
We've just released a two-font libre family derived from the Latin subset of our Indic fonts. Pan-European Latin character set plus diacritics for transliteration of South Asian languages, with smallcaps, ligating forms, assorted (non-tabular) numeral styles, etc.
![CastoroBannerSM](https://user-images.githubusercontent.com/5614500/84289721-b182ba00-aaf7-11ea-8fa5-6d38f9ffe973.jpg)

https://github.com/TiroTypeworks/Castoro

**Comment by davelab6** [comment]
I wonder if it's better to have this as part of the indics, rather than a Latin only font. How will the integrated latins in those font(s) be different to this, if at all?

**Comment by tiroj** [comment]
The Indics have a smaller Latin subset, basically just ANSI plus the diacritics for transliteraion of South Asian languages. The focus of the Indic fonts is really on the South Asian writing systems, as the names of the fonts suggest. Indeed, they really only contain Latin subsets for legacy technical reasons (needing to support at least one 8-bit codepage), and because we decided to match the glyph set of the fonts that HUP use to typeset the MCLI books, which included the transliteration diacritics in the same font as each Indic script.

I decided to release Castoro as a separate family as something specifically for European script languages, with its own identity and path. That path might involve things like Cyrillic and Greek or other things that it doesn't make sense to associate with a nominally ‘Indic’ set of fonts, or have to maintain across 8+ separate Indic script fonts.
